### PR TITLE
fix query logging

### PR DESCRIPTION
**Pull request by alexkorotkikh**
to avoid null to be printed

### CODE REVIEW

File: src/main/java/org/kairosdb/core/datastore/QueryMetric.java [hunk 1]
```diff
@@ -210,7 +210,7 @@ public void setQueryLoggingType(String type) {
 	}
 
 	public String getQueryLoggingType() {
-		return loggingType;
+		return loggingType == null ? "simple" : loggingType;
```
Comment (rajatparida86) [inline src/main/java/org/kairosdb/core/datastore/QueryMetric.java:213]: Should we do this in the setter method - `setQueryLoggingType `

Comment (alexkorotkikh) [reply src/main/java/org/kairosdb/core/datastore/QueryMetric.java:213]: This is needed in cases when setter method was never called and `loggingType` was left `null`. So doing it in the setter won't help

**Comment by rajatparida86** [comment]
👍

**Comment by alexkorotkikh** [comment]
:+1:

**Pull request merged**